packages/apps/desktop-viewer-test/src/frontend/components/widgets/PropertyGridWidget.tsx

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Orientation } from "@bentley/ui-core";
+import { ConfigurableCreateInfo, WidgetControl } from "@bentley/ui-framework";
+import * as React from "react";
+
+import SimplePropertiesComponent from "../Properties";
+
+/** A widget control for displaying the PropertyGrid React component */
+export class PropertyGridWidget extends WidgetControl {
```

Comment:
why is this needed? doesnt the base-viewer already have the property-grid?

Reply:
it's not. Copy leftover. I removed it